torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4105,6 +4107,11 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
            dtypesIfCUDA=all_types_and(torch.bool, torch.half, torch.bfloat16),
            sample_inputs_func=sample_inputs_atan2,
            ),
+    OpInfo('atan2',
+           dtypes=_DYNAMIC_DTYPES,
+           variant_test_name='1',
+           sample_inputs_func=sample_inputs_atan2,
+           ),
```

Comment:
TODO: Remove this. Just a sample for testing the failure.

Reply:
Question: if `dtypes` is `DYNAMIC_DTYPES` then do the CPU and CUDA have to implement the same dtypes?

Reply:
Right now `_DYNAMIC_DTYPES` just leads to failure to CI tests (test_ops.py) and queries the operator in `opinfo_helper.py`.
And doesn't set anything in the actual opinfo.
Though we can allow setting them here with function `get_supported_dtypes` as mentioned below.